source/content/guides/custom-upstream/01-custom-upstream.md

```diff
@@ -126,20 +126,38 @@ Support for Custom Upstreams requires that the externally hosted upstream reposi
 
-- **Public Upstream**: A repository that is open to all Pantheon users which contains a common codebase for new sites, like [Panopoly](https://github.com/populist/panopoly-drops-7).
+### Custom Upstream
 
-- **Repository**: A collection of files packaged in a single directory under version control.
+<p>A <dfn id="custom-upstream">custom upstream</dfn> is a repository restricted to members of an organization, containing a common codebase for new sites. This type of repository is a child repository to Pantheon's core upstreams (<a href="https://github.com/pantheon-systems/wordpress">WordPress</a>, <a href="https://github.com/pantheon-upstreams/drupal-composer-managed">Drupal (Latest Version)</a>, <a href="https://github.com/pantheon-systems/drops-7">Drupal 7</a>) and acts as a parent for site level repositories.</p>
 
-- **Remote Repository**: A central version control location, such as a repository residing on GitHub, Bitbucket, or GitLab.
+### Public Upstream
 
-- **Upstream Updates**: Code changes that are made once in a parent (upstream) repository, then applied "downstream" to child repositories. This is how Pantheon's one-click updates work.
+<p>A <dfn id="public-upstream">public upstream</dfn> is a repository that is open to all Pantheon users which contains a common codebase for new sites, like <a href="https://github.com/populist/panopoly-drops-7">Panopoly</a>.</p>
 
-- **Site Repository**: Child repository where upstream updates are applied and site specific customizations are tracked, similar to your site's codebase on Pantheon.
+### Repository
 
+<p>A<dfn id="repository">repository</dfn> is a collection of files packaged in a single directory under version control.</p>
+
+
+### Remote Repository
+
+<p>A <dfn id="remote-repository">remote repository</dfn> is the central version control location, such as a repository residing on GitHub, Bitbucket, or GitLab.</p>
+
+### Upstream Updates
+
+<p><dfn id="upstream-updates"> Upstream Updates</dfn> are code changes that are made once in a parent (upstream) repository, then applied "downstream" to child repositories. This is how Pantheon's one-click updates work.</p>
+
+### Site Repository
+
+<p>A <dfn id="site-repository">site repository</dfn> is a child repository where upstream updates are applied and site specific customizations are tracked, similar to your site's codebase on Pantheon.</p>
+
+### Framework
+
+<p>The <dfn id="upstream-framework">upstream framework</dfn> (or site framework) determines the server configuration for a given CMS. It includes things like Nginx configuration, the relevant CLI tool to install (Drush or WP-CLI), etc. This setting is <em>not visible</em> to users and once a framework has been set for a site (based on how the upstream was configured), it cannot be changed without <a href="https://docs.pantheon.io/guides/custom-upstream/switch-custom-upstream">changing the upstream</a> itself.</p>
```

Comment:
> This setting is <em>not visible</em> to users and once a framework has been set for a site (based on how the upstream was configured), it cannot be changed without <a href="https://docs.pantheon.io/guides/custom-upstream/switch-custom-upstream">changing the upstream</a> itself.

^ I don't think is accurate, because: 

* The site framework **is** visible from the site dashboard and also visible from the CLI `terminus site:info` 
  * the upstream framework isn't visible from a non-admin dashboard page but customers can see it from the CLI `terminus upstream:info` 
* A site's framework **cannot** be changed after site creation. Switching a site's upstream will only work when the current upstream and the new upstream have matching frameworks - it will not switch the site framework ever, and if you try to switch the site to an upstream with a different framework the workflow will fail. 
  * A custom upstream's framework **can** be changed after creation but only from the admin debug panel, so customers would have to contact support to do that  

![screencapture-dashboard-pantheon-io-sites-452cb6f4-4068-4275-bf10-fc3dd5670274-2024-12-04-15_13_20](https://github.com/user-attachments/assets/5c421397-ff55-4b7d-95f1-763ac6b01187)

<img width="850" alt="Screenshot 2024-12-04 at 3 35 05 PM" src="https://github.com/user-attachments/assets/51b7e07c-e31a-42df-856f-13ea59a9b105">